megamek/src/megamek/server/GameManager.java

```diff
@@ -2762,6 +2762,13 @@ private void determineTurnOrderIUI(GamePhase phase) {
         } else {
             entities = game.getEntitiesVector();
         }
+
+        String noInitiative = entities.stream().filter(e -> e.getInitiative().size() == 0).map(Object::toString).collect(Collectors.joining(";"));
```

Comment:
Can we split this into four lines for readability? filter, map, and collect would be better done on separate lines.